node/packages/aws-lambda-otel-extension/test/benchmark/performance.test..js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
```

Comment:
This file has a double `..` in the name. I think we just need to rename this to `performance.test.js` 😎

Reply:
Good find :+1. Fixing